.github/workflows/tests.yml

```
@@ -0,0 +1,91 @@
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22.13
+      - name: Install dependencies
+        run: npm install -g yarn && yarn install --frozen-lockfile
+      - name: Install Playwright Browsers
+        run: yarn playwright install --with-deps
+      - name: Wait for Vercel Preview Deployment
+id: get-vercel-url
```

actionlint (1.7.4)

unexpected key "id" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)